.semaphore/semaphore.yml

```diff
@@ -11,58 +11,88 @@ global_job_config:
   - name: docker-hub
   prologue:
     commands:
+      # make some room on the disk
+      - sudo rm -rf ~/.kiex ~/.phpbrew ~/.rbenv ~/.nvm ~/.kerl
       - echo $DOCKERHUB_PASSWORD | docker login --username "$DOCKERHUB_USERNAME" --password-stdin
       - checkout
       # Semaphore is doing shallow clone on a commit without tags.
       # unshallow it for GIT_VERSION:=$(shell git describe --tags --dirty --always) @ Makefile.common
       - git fetch --unshallow
 
 blocks:
+  - name: "Tests"
```

Comment:
Might consider naming this "Static checks / UT / FV" to make it more clear. "Tests" is pretty generic.

Reply:
I renamed it to "General Tests" because the job will have that name. There's 3 blocks, General Tests, System Tests, and Specialised Tests.